modules/representation_management/lib/representation_management/v0/pdf_constructor/form_2122.rb

```diff
@@ -0,0 +1,113 @@
+# frozen_string_literal: true
+
+module RepresentationManagement
+  module V0
+    module PdfConstructor
+      class Form2122 < RepresentationManagement::V0::PdfConstructor::Base
+        protected
+
+        def next_steps_page?
+          false
+        end
+
+        def template_path
+          Rails.root.join('modules',
+                          'representation_management',
+                          'lib',
+                          'representation_management',
+                          'v0',
+                          'pdf_constructor',
+                          'pdf_templates',
+                          '21-22.pdf')
+        end
+
+        # rubocop:disable Layout/LineLength
+        # rubocop:disable Metrics/MethodLength
+        def template_options(data)
```

Comment:
Same. I encourage you to refactor this method so you don't have to disable the cops:

```
        # rubocop:disable Layout/LineLength
        # rubocop:disable Metrics/MethodLength
```

Reply:
Done!